subprojects/build-init/src/main/groovy/org/gradle/buildinit/plugins/internal/maven/Maven2Gradle.groovy

```diff
@@ -160,7 +160,7 @@ class Maven2Gradle {
             mavenPublication.methodInvocation(null, "artifact", mavenPublication.propertyExpression("sourcesJar"))
         }
         if (testJarTaskGenerated) {
```

Comment:
⭕️ For consistency this should be called `testsJarTaskGenerated`.

Reply:
Done